rhosp7/templates/oval_5.11_templates/template_OVAL_service_disabled

```diff
@@ -33,12 +32,22 @@
     <linux:object object_ref="object_multi_user_target_for_%SERVICENAME%_socket_disabled" />
     <linux:state state_ref="state_systemd_%SERVICENAME%_socket_off"/>
   </linux:systemdunitdependency_test>
-
   <linux:systemdunitdependency_object id="object_multi_user_target_for_%SERVICENAME%_socket_disabled" comment="list of dependencies of multi-user.target" version="1">
     <linux:unit>multi-user.target</linux:unit>
   </linux:systemdunitdependency_object>
-
   <linux:systemdunitdependency_state id="state_systemd_%SERVICENAME%_socket_off" comment="%SERVICENAME% socket is not listed in the dependencies" version="1">
     <linux:dependency entity_check="none satisfy">%SERVICENAME%.socket</linux:dependency>
   </linux:systemdunitdependency_state>
+
+  <linux:systemdunitproperty_test id="test_service_not_running_%SERVICENAME%" check="all" check_existence="at_least_one_exists" comment="Test that the %SERVICENAME% service is not running" version="1">
```

Comment:
All `test_service_not_running_` should have `check_existence="any_exist", as it should be ok to not find the unit.